app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,9 @@
+class AdminController < ApplicationController
+  before_action :require_admin
+
+  def require_admin
```

Comment:
@castrolem what about using the same API Devise, `current_user` and `authenticate_user!`

Reply:
Sounds good to me. For now, these methods aren't verifying anything, because there's no `current_user`

Reply:
Actually, I just did the commit, but after my previous comment, I would vote to revert that change.